File: 프로그래머스/0/181909. 접미사 배열/접미사 배열.java
import java.util.*;
class Solution {
    public String[] solution(String my_string) {
        int len = my_string.length();
        String[] answer = new String[len];
        for(int i=0; i<answer.length; i++){
            answer[i]=my_string.substring(i);
            System.out.println(my_string.substring(i));
        }
        Arrays.sort(answer);
        return answer;
    }
}

File: 프로그래머스/0/181909. 접미사 배열/접미사 배열.py
def solution(my_string):
    answer = []
    my_string=list(my_string)
    my_string.reverse()
    my_string2=''.join(my_string)
    print(my_string2)

    answer.append(my_string[0])
    for i in range(0, len(my_string)-1):
        answer.append(my_string[i+1]+answer[i])
    answer.sort()
    return answer